Assets/Scripts/Rooms/MapLayoutManagerStepThrough.cs

```diff
@@ -0,0 +1,441 @@
+﻿using System.Collections;
```

Comment:
This looks like code I made?

Reply:
yea something beyond my git comprehension is happening,
I git pulled the 'main' branch into this branch using the "ours" recursive strategy